src/main/java/org/jboss/windup/operator/controller/WindupController.java

```diff
@@ -0,0 +1,80 @@
+package org.jboss.windup.operator.controller;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClientException;
+import io.fabric8.kubernetes.client.Watcher;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import lombok.extern.java.Log;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jboss.windup.operator.model.WindupResource;
+import org.jboss.windup.operator.model.WindupResourceDoneable;
+import org.jboss.windup.operator.model.WindupResourceList;
+import org.jboss.windup.operator.util.WindupDeployment;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@Log
+@ApplicationScoped
+public class WindupController implements Watcher<WindupResource> {
+	@Inject
```

Comment:
_Comment to further review the code, NOT mandatory for this PR_
This class is "Tab" indented: please change it to be "4 spaces" indented like `WindupDeploymentController`.
Then please apply the same indentation to change to other classes to have a consistent indentation.